### PR TITLE
Features/mobile menu javascript

### PR DESCRIPTION
**Pull request by Wahaj-Ali**
In this branch
- implemented JavaScript for the "hamburger icon" in the mobile version of the desktop. 
- created scripts.js to add JavaScript code.
- Restyled the mobile version to adjust according to the changes made.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,139 +13,154 @@
...
-        <nav class="desktop_headicons">
-            <a class="header_btn" href="">Portfolio</a>
-            <a class="header_btn" href="#about_text">About</a>
-            <a class="header_btn" href="#full_name">Contact</a>
-            <a class="header_btn" href=""><img id="mail" src="attributes/mailicon.png" alt="mailicon" /></a>
+    <header>
+        <nav class="desktop-nav">
+            <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
+            <ul class="main-menu">
+                <li class="nav-item"><a class="header-btn" href="">Portfolio</a></li>
```
Comment (ha-manel) [inline index.html]: - [ ] This link is supposed to take us to the works section. Kindly update the `href` accordingly 👍

File: index.html [hunk 2]
```diff
@@ -13,139 +13,154 @@
...
-        <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
-        <nav><img class="union" src="attributes/union.png" alt="union_icon" /></nav>
-        <nav class="desktop_headicons">
-            <a class="header_btn" href="">Portfolio</a>
-            <a class="header_btn" href="#about_text">About</a>
-            <a class="header_btn" href="#full_name">Contact</a>
-            <a class="header_btn" href=""><img id="mail" src="attributes/mailicon.png" alt="mailicon" /></a>
+    <header>
+        <nav class="desktop-nav">
+            <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly hide the logo from inside the mobile menu to match the design in Figma. Also, kindly add `border-bottom` to all links to match the design 👍 

## Your design:
image

## Figma design:
image

File: index.html [hunk 3]
```diff
@@ -13,139 +13,154 @@
...
-    <header class="container-header">
-        <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
-        <nav><img class="union" src="attributes/union.png" alt="union_icon" /></nav>
-        <nav class="desktop_headicons">
-            <a class="header_btn" href="">Portfolio</a>
-            <a class="header_btn" href="#about_text">About</a>
-            <a class="header_btn" href="#full_name">Contact</a>
-            <a class="header_btn" href=""><img id="mail" src="attributes/mailicon.png" alt="mailicon" /></a>
+    <header>
+        <nav class="desktop-nav">
```
Comment (ha-manel) [inline index.html:17]: [Optional]
- [ ] It's better if we can't scroll when we open the mobile menu 👍 

Animation

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi Team👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

## Optional:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Wahaj-Ali 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀   

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  Kindly make sure that for [pair programming](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) projects, both partners have commits, currently, there are no commits from your partner 👍🏿 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**